app/assets/javascripts/modules/FilterTable.js

```diff
@@ -3,19 +3,20 @@
  *
  * Requires an element to have a data-dough-component="FilterTable" attribute and an ID.
  *
- * Makes use of ListJS — see listjs.com.
+ * Makes use of jquery.fastLiveFilter.
  * See test fixture for sample markup - /spec/js/fixtures/FilterTable.html
  */
 
-define(['jquery', 'DoughBaseComponent', 'List'],
-       function($, DoughBaseComponent, List) {
+define(['jquery', 'jqueryFastLiveFilter', 'DoughBaseComponent'],
+       function($, jqueryFastLiveFilter, DoughBaseComponent) {
   'use strict';
 
   var FilterTableProto,
       defaultConfig = {
-        filterTargetClass: 'js-filterable',
-        filterFieldClass: 'js-filter-field',
-        filterListClass: 'js-filter-rows'
+        filterTargetClass: '.js-filterable',
+        filterFieldClass: '.js-filter-field',
+        filterListClass: '.js-filter-rows',
+        filterWrapperClass: '.js-filter-wrapper',
```

Comment:
Since we've changed these from classes to selectors — can we rename these according to `s/Class/Selector/`?